### PR TITLE
Added a text based spy disguise decoration

### LINKED ISSUE #12225: Change spy-tag
noticed that the lettering of the "fake buildings" actually fit also on Infantry ... so i suggest we use a tag that actually is readable instead of the "eye?" 
this would actually fit very well , since buildings aswell as infantry can actually be fake ... so the lettering would make it consistent 

spytag

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/12225.

### CODE REVIEW

**Comment by pchote** [comment]
Please lets wait until after #17808 to avoid rebase nightmares.

**Comment by Mailaender** [comment]
Rebased.

File: mods/ra/sequences/misc.yaml [hunk 1]
```diff
@@ -128,6 +128,9 @@ pips:
 		Tick: 300
 		Offset: 0, -6
 
+disguise:
```
Comment (pchote) [inline mods/ra/sequences/misc.yaml]: Please define this as `tag-spy: tag-spy` after `tag-primary` inside `pips` and rename the file to `tag-spy.shp`. There's no reason to break convention with the fake and primary tags here.

**Review by pchote — COMMENTED** [review]
The tag clashes with the selection box at the Far


and Furthest


zoom levels. Please set a Y offset to put it above the health bar instead.

Please also fix the sprite to use the proper black colour.

**Comment by Mailaender** [comment]
How do you set the zoom level to far and furthest? The UI only shows me close and medium.

**Comment by pchote** [comment]
Far requires a vertical screen resolution > 900px, and Furthest > 1300px. Strictly speaking, its the ratio of the UI scale to the viewport size that matters - you can achieve the same result at closer world zoom by forcing the game to use a large UI scale.

**Comment by Mailaender** [comment]
Updated.

**Review by Smittytron — APPROVED** [review]
LGTM

**Review by obrakmann — APPROVED** [review]
:+1: much better than the icon

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/db8fdbce085e482d55553e46d71b13bfe6ddecd0)

**Pull request merged**